apis/spotify.py

```diff
@@ -12,6 +14,26 @@
 )
 
 
+def get_wiki_url(title: str, content: str, creator: str='') -> str:

```

Comment:
don't leave empty strings as the default it'd be better to make it an `Optional[str]` at that point and just do a check to fill in the create value with a default symbol like you have.

or just make the param just have '_', this will remove a few lines of code.
`def get_wiki_url(title: str, content: str, creator: str='_')`

Also can you set '_' as a DEFAULT_CREATOR var at the top